_posts/2016-06-28-position_sheffield.md

```diff
@@ -1,6 +1,6 @@
 ---
 layout: post
-title: Full time position, University of Sheffield, United Kingdom
+title: Full time position has expired, University of Sheffield, United Kingdom
```

Comment:
I suggest to leave the title as is - otherwise it may be misunderstood like someone left because their contract had expired.